installer/manifests/keptn/charts/control-plane/templates/mongodb-datastore.yaml

```diff
@@ -63,8 +63,8 @@ spec:
           value: "{{ .Values.prefixPath }}"
         - name: MONGODB_HOST
           value: '{{ .Values.mongodb.host }}'
-        - name: MONGODB_DATABASE
-          value: 'keptn'
+        - name: MONGO_DB_NAME
+          value: {{ .Values.mongodb.dbName | default "keptn"}}
```

Comment:
Do you really need the default here, if the name is already provided in the values file directly?

Reply:
the name should be provided in the values.yaml file, I just added the default to be safe